app/src/main/java/org/xwiki/android/sync/auth/RegistrationActivity.kt

```diff
@@ -0,0 +1,19 @@
+package org.xwiki.android.sync.auth
+
+import android.os.Bundle
+import android.webkit.WebView
+import androidx.appcompat.app.AppCompatActivity
+import org.xwiki.android.sync.R
+import org.xwiki.android.sync.URL_FIELD
+import org.xwiki.android.sync.utils.openLink
+
+class RegistrationActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_registration)
+        val webView = findViewById(R.id.webview) as WebView
```

Comment:
It is better to use syntax like `findViewById<WebView>` instead of `as WebView` cast

Reply:
Updated.